internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -90,17 +90,33 @@ private ConfigKeys() {
   public static final String KAFKA_MIN_LOG_COMPACTION_LAG_MS = "kafka.min.log.compaction.lag.ms";
 
   /**
-   * The min.isr property to be set at topic creation time. Will not modify already-existing topics.
+   * The min. in sync replicas property to be set at topic creation time. Will not modify already-existing topics. This
+   * config only applies to store version topics.
    *
    * If unset, will use the Kafka cluster's default.
    */
   public static final String KAFKA_MIN_ISR = "kafka.min.isr";
 
   /**
-   * The replication factor to set for real-time buffer topics and store-version topics, at topic creation time.
+   * Same as KAFKA_MIN_ISR except it applies to real-time buffer topics.
+   */
+  public static final String KAFKA_MIN_ISR_RT_TOPICS = "kafka.min.isr.rt.topics";
+
+  /**
+   * Same as KAFKA_MIN_ISR except it applies to admin topics.
+   */
+  public static final String KAFKA_MIN_ISR_ADMIN_TOPICS = "kafka.min.isr.admin.topics";
+
+  /**
+   * The replication factor to set for store-version topics only, at topic creation time.
    */
   public static final String KAFKA_REPLICATION_FACTOR = "kafka.replication.factor";
 
+  /**
+   * Same as KAFKA_REPLICATION_FACTOR except it applies to real-time buffer topics.
```

Comment:
Idk if I'm a fan of this linked list style comment.  I'd say just put "The replication factor to set for real-time buffer topics".

Reply:
As an example, another way to interpret this sentence might be that the two variable values are the same.